pxr/imaging/hdx/presentTask.cpp

```diff
@@ -103,11 +88,7 @@ HdxPresentTask::Execute(HdTaskContext* ctx)
         // framebuffer contents.
         // Eg. This allows us to render with HgiMetal and present the images
```

Comment:
This comment could be improved as well, since this change is no longer focused specifically on GL interop. Something like ~

```
Accordingly, we may render with an Hgi backend that differs from the rendering system used by a particular application. for example, using Vulkan to render, and OpenGL to display in an application.
```